app/keepers/keepers.go

```diff
@@ -260,6 +260,19 @@ func (appKeepers *AppKeepers) InitNormalKeepers(
 
 	appKeepers.EpochsKeeper = epochskeeper.NewKeeper(appCodec, appKeepers.keys[epochstypes.StoreKey])
 
+	txFeesKeeper := txfeeskeeper.NewKeeper(
+		appCodec,
+		appKeepers.AccountKeeper,
+		appKeepers.BankKeeper,
+		appKeepers.EpochsKeeper,
+		appKeepers.keys[txfeestypes.StoreKey],
+		appKeepers.GAMMKeeper,
+		appKeepers.GAMMKeeper,
```

Comment:
@p0mvn why do we need to do use `appKeepers.GAMMKeeper` twice here?

Reply:
I'm not sure - I think we should have grouped this into a single parameter and consolidate the two interfaces (`SpotPriceCalculator` and `GammKeeper`) into a single one

If you you'd like, please make an issue to fix this. If no objections from others, we can work on this

Reply:
Gotchu i'll look into it and try to fix it!